### PR TITLE
fix: renamed series in pie charts tooltip

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

In [this community slack thread](https://posthogusers.slack.com/archives/C01GLBKHKQT/p1663035431847969) the user reports that for a renamed pie chart series we don't show any breakdown value in the tooltip. So you can't see what that section of the chart is for

Screenshot 2022-09-15 at 10 39 24

## Changes

Updates pie charts to use our external tooltip rendering

This is slightly over duplicated. But there's a pie chart refresh coming up in #11433 and extracting the tooltip as a function was a little tricky

## How did you test this code?

running it locally and seeing it work

### CODE REVIEW

**Comment by guidoiaquinti** [comment]
Thank you @pauldambra ! 🙇

**Review by benjackwhite — APPROVED** [review]
Looks good. One follow up we should do is add a debounce on the tooltip position change as it moves super slugishly with the css animation.

File: frontend/src/scenes/insights/views/LineGraph/LineGraph.tsx [hunk 1]
```diff
@@ -508,18 +514,83 @@ export function LineGraph_({
...
+                                            groupTypeLabel={
+                                                labelGroupType === 'people'
+                                                    ? 'people'
+                                                    : labelGroupType === 'none'
+                                                    ? ''
+                                                    : aggregationLabel(labelGroupType).plural
+                                            }
+                                            {...tooltipConfig}
+                                        />
+                                    </>,
```
Comment (mariusandra) [inline frontend/src/scenes/insights/views/LineGraph/LineGraph.tsx:584]: Nit: you could remove the `<>` for less indented code.

**Review by mariusandra — APPROVED** [review]

**Comment by pauldambra** [comment]
Added a short debounce. Great call @benjackwhite 🙌

**Pull request merged**